boards/x86/intel_rpl/doc/index.rst

```diff
@@ -44,7 +44,7 @@ Build Zephyr application
 
```

Comment:
Should we add `S` also here and in all other places ?

Reply:
> Should we add `S` also here and in all other places ?

Yes, I think we should. I missed it since I only grepped for rpl_crb when looking for places to change.

Reply:
Updated now